test/artefacts/marriage-abroad/ethiopia/ceremony_country/partner_local/same_sex.txt

```diff
@@ -0,0 +1,6 @@
+Civil partnership in Ethiopia
+
+It’s not possible to get legal recognition for your same-sex relationship in Ethiopia.
+
+
+
```

Comment:
There's extra blank trailing lines in these artefacts but not in the content files.  Is this correct?

Reply:
This didn't seem to affect the regression test results but I've removed the whitespace all the same, probably a side-effect of copy-n-paste.